presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -831,22 +830,20 @@ protected Void visitCreateTable(CreateTable node, Integer indent)
                 builder.append("\nCOMMENT " + formatStringLiteral(node.getComment().get()));
             }
 
-            appendTableProperties(builder, node.getProperties());
+            builder.append(formatTableProperties(node.getProperties()));
 
             return null;
         }
 
-        private void appendTableProperties(StringBuilder builder, Map<String, Expression> properties)
+        private String formatTableProperties(List<TableProperty> properties)
         {
-            if (!properties.isEmpty()) {
-                builder.append("\nWITH (\n");
-                // Always output the table properties in sorted order
-                String propertyList = ImmutableSortedMap.copyOf(properties).entrySet().stream()
-                        .map(entry -> INDENT + formatName(entry.getKey()) + " = " + formatExpression(entry.getValue(), parameters))
-                        .collect(joining(",\n"));
-                builder.append(propertyList);
-                builder.append("\n").append(")");
+            if (properties.isEmpty()) {
+                return "";
             }
+            String propertyList = properties.stream()
+                    .map(element -> formatExpression(element.getName(), parameters) + " = " + formatExpression(element.getValue(), parameters))
```

Comment:
This changes the formatting and will break the tests (which are correct). Please preserve the original formatting.

Reply:
A separate PR has been merged to address the ordering. ShowQueryRewrites now sorts the properties rather than SqlFormatter.